packages/blade/src/components/Icons/useIconProps/useIconProps.tsx

```diff
@@ -14,8 +14,8 @@ const iconSize = {
 } as const;
 
 function useIconProps({
-  size,
-  color,
+  size = 'medium',
+  color = 'feedback.icon.neutral.lowContrast',
```

Comment:
Where did we decide this default color? is it from design?

Reply:
I decided it myself. Probably we can default to `currentColor` 🤔

Reply:
Let's ask rastogi once, design main kya hain.

Reply:
Updated it to `surface.text.normal.lowContrast` as per design